docs/source/index.rst

```diff
@@ -6,12 +6,88 @@
 PySAGES: Python Suite for Advanced General Ensemble Simulations
 ===============================================================
 
-No compromises in usability and speed for advanced sampling methods!
+
+.. only:: html
+
+    |Github Actions CI|
+    |Github Actions CI Docker|
+    |Documentation Status|
+
+    .. |Documentation Status| image:: https://readthedocs.org/projects/pysages/badge/?version=latest
+        :target: https://pysages.readthedocs.io/en/latest/?badge=latest
+    .. |GitHub Actions CI| image:: https://github.com/SSAGESLabs/PySAGES/actions/workflows/ci.yml/badge.svg?branch=main
+        :target: https://github.com/SSAGESLabs/PySAGES/actions/workflows/ci.yml
+    .. |GitHub Actions CI Docker| image:: https://github.com/SSAGESLabs/PySAGES/actions/workflows/docker-ci.yml/badge.svg?branch=main
+        :target: https://github.com/SSAGESLabs/PySAGES/actions/workflows/docker-ci.yml
+
+
+Molecular dynamics (MD) simulations are powerful tools to investigate the static and dynamic properties of a given system.
+However, even with modern computer architecture and the fastest simulation software, computation time is limited and valuable.
+As a result, exploring a system by unbiased MD is insufficient to obtain good statistics, especially if the free-energy landscape is separated by high barriers.
+To address this well know drawback enhanced-sampling methods have been established.
```

Comment:
*known

@nrouizem  The "drawback" this sentence refers to is what the previous sentence says. Maybe for better clarity: "To address this well known limitation/drawback of unbiased MD, enhanced-sampling methods have been established." ?